src/go/rdctl/cmd/shell.go

```diff
@@ -53,6 +54,16 @@ var shellCmd = &cobra.Command{
 	},
 }
 
+type cmdWriter struct {
+	result string
+	write  func(bytes []byte)
+}
+
+func (writer *cmdWriter) Write(bytes []byte) (int, error) {
+	writer.result += string(bytes)
+	return len(bytes), nil
+}
+
```

Comment:
The `cmdWriter` code is no longer used and should be deleted

Reply:
gone